Makefile

```diff
@@ -1,9 +1,8 @@
 SHELL=/bin/bash
 export PATH := ./bin:./venv/bin:$(PATH)
 
-ifndef ELASTIC_VERSION
+# Determine the version to build. Override by setting ELASTIC_VERSION env var.
 ELASTIC_VERSION := $(shell ./bin/elastic-version)
```

Comment:
I kinda buried the explanation in [this commit message](https://github.com/elastic/elasticsearch-docker/pull/101/commits/98b7d93ce5216aa68ce0121706f9536804832b88). I should have updated the PR body. Sorry.